cluster-up/cluster/ephemeral-provider-common.sh

```diff
@@ -194,6 +194,10 @@ function _add_common_params() {
         params=" --deploy-multus $params"
     fi
 
+    if [ "$KUBEVIRT_WITH_DNC" == "true" ]; then
+        params=" --deploy-dnc $params"
+    fi
```

Comment:
Since this configuration is a sub-configuration of `KUBEVIRT_WITH_CNAO`, please consider moving it under the condition testing `KUBEVIRT_WITH_CNAO`'s value.